slackboard/notify.go

```diff
@@ -1,14 +1,18 @@
 package slackboard
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"strings"
 	"time"
 )
 
+var (
+	SlackPostMessageAPIURL = `https://slack.com/api/chat.postMessage`
+)
```

Comment:
How about making it const?

Reply:
This variable will be changed when testing. ([example](https://github.com/cubicdaiya/slackboard/pull/17/files/0eb0a1d161c0d118b6eea934a3c2ad6d315e0d0b#diff-1a63b1c2cd924c568acc7d8f0255aa7fR95))
This technique is used in [nlopes/slack](https://github.com/nlopes/slack/tree/v0.6.0/slacktest)